org/mozilla/jss/netscape/security/x509/X509Key.java

```diff
@@ -210,6 +213,8 @@ static X509Key buildX509Key(AlgorithmId algid, byte[] key)
         } catch (ClassNotFoundException e) {
         } catch (InstantiationException e) {
         } catch (IllegalAccessException e) {
```

Comment:
I think we still want `IllegalAccessException`  to trigger the `IOException`; `InvocationTargetException` maybe as well, I'm not sure. But these are the only ones that might happen; the others shouldn't happen at all. 

`NoSuchMethodException` shouldn't happen since we're getting the constructor so I'm fine suppressing that exception like the ones above.